src/custom/state/ethFlow/hooks.ts

```diff
@@ -0,0 +1,39 @@
+import { useAtomValue } from 'jotai/utils'
+import { isUserNativeEthFlow } from './atoms'
+
+import { useState } from 'react'
+import { useSwapState } from 'state/swap/hooks'
+import { useCurrency } from 'hooks/Tokens'
+import { Field } from 'state/swap/actions'
+import { useDetectNativeToken } from 'state/swap/hooks'
+
+export function useIsUserNativeEthFlow() {
+  return useAtomValue(isUserNativeEthFlow)
```

Comment:
As mentioned, you can use the base atom directly, no need for this extra selector layer

Reply:
i just preferred the name of the function, but i see your point, do you feel strongly about the proposed change?